packages/server/lib/project-base.ts

```diff
@@ -426,6 +433,16 @@ export class ProjectBase extends EE {
 
       onStudioInit: async () => {
         if (this.spec && this.ctx.coreData.studio?.protocolManager) {
+          const canAccessStudioLLM = await this.ctx.coreData.studio?.canAccessStudioLLM({
+            family: this.browser.family,
+            name: this.browser.name,
+            channel: this.browser.channel,
+          }) ?? false
```

Comment:
Can we just pass in the browser?

```suggestion
          const canAccessStudioLLM = await this.ctx.coreData.studio?.canAccessStudioLLM(this.browser) ?? false
```

Reply:
Potentially. There's more stuff on there so I was trying to be somewhat explicit about what I was sending.

Reply:
Since I'm using the Cypress types now, I can do this.